protocol/app/msgs/normal_msgs_test.go

```diff
@@ -125,6 +125,9 @@ func TestNormalMsgs_Key(t *testing.T) {
 		"/dydxprotocol.sending.MsgWithdrawFromSubaccount",
 		"/dydxprotocol.sending.MsgWithdrawFromSubaccountResponse",
 
+		// ica
```

Comment:
nit: can we keep this alphabetical? would prefer adding this at the end after all the `ibc...`

Reply:
I tried to make the actual string value alphabetical, since this one also starts with `/ibc.applications.....`

Updated the comment for consistency!